app/webpacker/styles/filter_toggle_button.scss

```diff
@@ -27,17 +27,26 @@ $shadow-color: #0b0c0c;
 }
 
 .moj-action-bar__filter {
+  float: none;
+  margin-top: 30px;
+
   @include govuk-media-query($from: desktop) {
     display: none;
   }
-}
 
-.filter-toggle-button {
-  float: right;
-  margin-bottom: govuk-spacing(1);
+  .govuk-button {
+    margin: 0;
+  }
```

Comment:
I might be cautious about styling on this class, which is an unrelated component.